client/scripts/index.js

```diff
@@ -85,8 +90,7 @@ requirejs(['responsive-sketchpad/sketchpad'],
 
     }
 )
+
 const socket = io()
 
-socket.on('response', data => {
-    console.log("Server says: " + JSON.stringify(data))
-})
+socket.on('new-image', update_image_from_server)
```

Comment:
does this actually call update_image_from_server with `data`?
I would have done:
```
socket.on('new-image', data => {
    update_image_from_server(data)
})

```